projects/discussion-ui/src/lib/components/tag-all-discussion/tag-all-discussion.component.ts

```diff
@@ -0,0 +1,138 @@
+import { Component, OnInit } from '@angular/core'
+import { NSDiscussData } from '../../models/discuss.model'
+import { Router, ActivatedRoute } from '@angular/router'
+import { DiscussionService } from '../../services/discussion.service';
+/* tslint:disable */
+import _ from 'lodash'
+import { Subscriber, Subscription } from 'rxjs';
+import { ConfigService } from '../../services/config.service';
+import * as CONSTANTS from './../../common/constants.json';
+import { DiscussUtilsService } from '../../services/discuss-utils.service';
+
+@Component({
+  selector: 'lib-tag-all-discussion',
+  templateUrl: './tag-all-discussion.component.html',
+  styleUrls: ['./tag-all-discussion.component.scss']
+})
+export class TagAllDiscussionComponent implements OnInit {
+
+  tagName!: any
+  similarPosts!: any
+  queryParam: any
+  fetchSingleCategoryLoader = false
```

Comment:
The loader is not added in the HTML component. Then what's the use of this variable?

Reply:
loader is added